Cargo.toml

```diff
@@ -22,16 +22,16 @@ panic = "unwind"
 
 # Uncomment these lines for local development paths
 
-#[patch.'https://github.com/HorizenOfficial/ginger-lib.git']
-#algebra = { path = '../ginger-lib/algebra' }
-#algebra-utils = { path = '../ginger-lib/algebra-utils' }
-#primitives = { path = '../ginger-lib/primitives' }
-#r1cs-core = { path = '../ginger-lib/r1cs/core' }
-#r1cs-crypto = { path = '../ginger-lib/r1cs/gadgets/crypto' }
-#r1cs-std = { path = '../ginger-lib/r1cs/gadgets/std' }
-#
-#[patch.'https://github.com/HorizenLabs/marlin.git']
-#marlin = { path = '../marlin' }
-#
-#[patch.'https://github.com/HorizenLabs/poly-commit.git']
-#poly-commit = { path = '../poly-commit' }
+[patch.'https://github.com/HorizenOfficial/ginger-lib.git']
```

Comment:
Don't forget to comment these lines to avoid build errors

Reply:
Thanks. We'll remove this workaround in the end.

Reply:
done